Source/WTF/wtf/spi/cocoa/XTSPI.h

```diff
@@ -36,3 +36,16 @@ enum {
 };
 
 typedef uint32_t XTScope;
+
+enum {
+    kXTOptionsDefault                   = 0,
+    kXTOptionsPreferAppleSystemFonts    = 1 << 0,
+    kXTOptionsRemoveDuplicateFonts      = 1 << 1,
+    kXTOptionsRemoveInvisibleFonts      = 1 << 2,
+    kXTOptionsRestrictToPostScriptName  = 1 << 3,
+    kXTOptionsDoNotSortResults          = 1 << 4,
+    kXTOptionsIncludeDisabledFonts      = 1 << 5,
+    kXTOptionsDisableFontProviders      = 1 << 16,
+    kXTOptionsDisableAutoActivation     = 1 << 17
+};
```

Comment:
I think you drop the unused options

```suggestion
enum {
    kXTOptionsDoNotSortResults          = 1 << 4
};
```

Reply:
That is a good point, fixed in latest patch.

Thanks for reviewing!